users/tests.py

```diff
@@ -1,3 +1,207 @@
-from django.test import TestCase
+import bcrypt
+import jwt
 
-# Create your tests here.
+from django.test              import TestCase, Client
+
+from .models                  import User
+from .utils                   import Login_decorator
+from mylittletrip.my_settings import JWT_ALGORITHM, SECRET_KEY
+
+class SighUpTestCase(TestCase):
+    def setUp(self):
+        self.URL = '/users/sigh_up'
+        self.client = Client()
+        hashed_password = bcrypt.hashpw('12345678'.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+        User.objects.create(name = 'Dummy', email = 'dummy@naver.com', password = hashed_password, phone_number = '01012341234')
+    
+    def tearDown(self):
+        pass
+
+    def test_post_success(self):
```

Comment:
```suggestion
    def test_sign_up_post_success(self):
```
물론 클래스로 SignUpTest라는 것을 밝혀놓으시긴 하였지만, 테스트 케이스 함수의 이름은 어떤 경우를 테스트하기 위함인지 명확히 알려줄 수 있어야 합니다 :-) 어떤 작업인지 정확하게 알 수 있도록 다른 함수 이름도 모두 좀 더 구체적인 것으로 바꿔주세요~

Reply:
다음부터 구체적으로 명시하겠습니다